internal/server/recon/recognize_test.go

```diff
@@ -390,3 +390,173 @@ func TestRankAndTrimCandidates(t *testing.T) {
 		}
 	}
 }
+
+func TestGetId2Span(t *testing.T) {
+	for _, c := range []struct {
+		query string
+		want  map[string]map[string]struct{}
+	}{
+		{
+			"entities1, Entities^2 , and entities,^2",
+			map[string]map[string]struct{}{
+				"entities1": {
```

Comment:
Same as other tests, can you please add a comment each?

Reply:
done